README.md

```diff
@@ -27,5 +27,94 @@ MODEL MANAGERS
 QUERY BUILDERS
 - QueryBuilder - Class to support creation of cypher queries to work with data in Neo4j  
 
-Dependencies:
+## End-to-end data loading and reshaping example
+
+The example code below runs through a use case of data loader, model manager, model applier and data provider.
+
+#### Importing the data 
+
+The call to FileDataLoader in the code below connects to the database using environment variables for your host and log-in credentials. Alternatively you can use the format `FileDataLoader(host="bolt://...",credentials=("username","password"))` to connect. The call to clean_slate() empties the database, and then load_file reads in your data from the specified filepath.
+
+```python
+from model_managers import ModelManager
+from data_loaders import FileDataLoader
+from model_appliers import ModelApplier
+from data_providers import DataProvider
+
+fdl = FileDataLoader()
+fdl.clean_slate()
+fdl.load_file(
+    folder='examples/data/', 
+    filename='Record.csv'
+)
+```
+
+Now the data will be populated within your database, but it won't be connected in a meaningful way. We have \`Source Data Column\` nodes containing information about the columns of our data - Study, Subject, Age and Sex, and \`Source Data Row\` nodes containing information of the two rows. The following code creates a trivial model(Classes and Relationships) from the data using ModelManager.
+
+```python
+mm = ModelManager()
+mm.create_model_from_data()
+```
+Here we can see we now have class and relationship nodes, illustrating connections between Study, Subject, Age and Sex. The red nodes below indicate relationship nodes, while the blue indicate class nodes.
+
+![modelmanager example](examples/data/modelmanager_example.png)
+
+Now using that class-relationship model we built using ModelManager, we can refactor our data and extract entities of the defined classes into separate nodes. In the code below we use `refactor_all` to do this, and we can see in the image that age values of 40 and 50 have been extracted into their own nodes. 
+
+```python
+ma = ModelApplier(mode="schema_CLASS")
+ma.refactor_all()
+
+```
+
+![modelapplier example](examples/data/modelapplier_example.png)
+
+Now we have our data set up, we can call it back using DataProvider. Here we call back the Age class. As we have not specified any relationships, we must set `infer_rels=True`.
+
+```python
+dp.get_data_generic(["Age"],infer_rels=True) 
+```
+
+```
+            _id_Age  Age.rdfs:label  
+    0            175415      30
+    1            175416      40
```

Comment:
@shannonhaughton 
Thanks.
Calling back just Age is not quite a good example I would say.
did you try dp.get_data_generic(["Subject", "Record", "Age"], infer_rels=True) 
?

also I would provide an example with get_data_generic options providing the columns as they where in the source data (return_propname=False), and either exclude neo4j ids from the output setting return_nodeid=False or elaborate on what columns _id_... mean